opentelemetry-userevents-exporter/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "opentelemetry-userevents-exporter"
+description = "OpenTelemetry-Rust exporter to userevents"
+version = "0.1.0"
+edition = "2021"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-userevents-exporter"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-userevents-exporter"
+readme = "README.md"
+rust-version = "1.68.0"
+keywords = ["opentelemetry", "log", "trace", "user_events"]
+license = "Apache-2.0"
+
+[features]
+logs = ["dep:async-trait", "opentelemetry_sdk/logs"]
+default = ["logs"]
```

Comment:
Is there a point to having a features section if the default encapsulates all features?

Reply:
it would allow opting out via `default-features = false` https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature

The question I guess is does this crate still make sense without that feature enabled?